languages/java/exercises/concept/classes/.docs/after.md

```diff
@@ -0,0 +1,101 @@
+The primary object-oriented construct in Java is the _class_, which is a combination of data ([_fields_][fields]), also known as instance variables, and behavior ([_methods_][methods]). The fields and methods of a class are known as its _members_.
+
+Access to members can be controlled through access modifiers, the two most common ones being:
+
+- [`public`][public]: the member can be accessed by any code (no restrictions).
+- [`private`][private]: the member can only be accessed by code in the same class.
+
+It is customary to specify an access modifier for all members. If no access modifier is specified, it will default to `private`.
```

Comment:
In Java if no access modifier is specified, the default one is `package visibility`.
So the member is visible to all classes defined into the same package